### DISCUSSION TITLE
Can we store the bookmarks locally with the project source code and sync it via git?

### DISCUSSION THREAD

**Discussion by fripSide** [alefragnani/vscode-bookmarks#605] [Q&A] [answered]
I need to develop with different computers and I want sync my bookmarks of a project.

**Comment by alefragnani** [comment, marked as answer]
Hi @fripSide ,

Simply update your settings and use `bookmarks.saveBookmarksInProject: true`. The extension will store the bookmarks in `.vscode/bookmarks.json` file. 

Hope this helps

**Reply by fripSide** [reply]
Thanks very much. This feature is extremely useful.
